src/main/resources/lang/hu.json

```diff
@@ -0,0 +1,54 @@
+{
+  "localization": {
+    "language": "Hungarian (HU)",
+    "translator": {
+      "name": "Ben Woodworth",
+      "from": "benwoodworth.net"
```

Comment:
Lines 5 and 6 say who translated the file, so you should put your own information.